depthcharge/components/encoders.py

```diff
@@ -28,14 +28,14 @@ def __init__(self, dim_model, min_wavelength=0.001, max_wavelength=10000):
             base = min_wavelength / (2 * np.pi)
```

Comment:
**question:** Should line 24 use `math.ceil` instead of `int` to match to the $\leq$?

Reply:
Good catch!